src/main/java/me/zeroeightsix/kami/util/astaralg/MyAstarNode.java

```diff
@@ -0,0 +1,73 @@
+package me.zeroeightsix.kami.util.astaralg;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.Minecraft;
+import net.minecraft.init.Blocks;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.HashSet;
+
+/**
+ * Created by fred41 on 18/07/2020.
+ */
+public class MyAstarNode {
+    public boolean blocked;
+    public double g = 0;
+    public MyAstarNode pred;
+    public static HashSet<Block> noclipblocks = new HashSet<>();
+
+    static {
```

Comment:
should this really be done every time? You can declare a HashSet with stuff already inside it, no?

Reply:
i don't know a way to initialize static HashSets better. the static block and the constructor is only executed once if im not mistaken. it can be moved somewhere else if you want.

Reply:
I'll rewrite it in kotlin then